week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,48 +1,58 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
-function mood() {
-  let isHappy = true;
+let isHappy = true;
 
-  if (isHappy) {
+function mood(isHappy) {
+
+  if (isHappy == true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
-function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
 
-  if (isBigEnough) {
+let num = 10;
+let isBigEnough = num;
+
+
+function greaterThan10(isBigEnough) {
+  
+  if (isBigEnough = num) {
```

Comment:
What this exercise wants you to do is use the `>` operator to compare the number inside the argument `isBigEnough` with the number `10`.

Could you give this a try?